src/App.php

```diff
@@ -147,70 +159,196 @@ public function __construct($defaults = [])
+            $errName = '';
+            
+            $ignoreError = false;
+            
+            $ignoreDeprecated = true;
+            $ignoreNotice = !$isDebugActive;
+            $ignoreWarning = !$isDebugActive;
+            
+            /**
+             * if we ignore some errors, this doesn't mean that will not log it
+             *
+             * when debug is active, ignored error will be logged
+             */
+            $logIgnoredError = $isDebugActive;
+            
+            switch ($errno) {
+                
+                case E_ERROR:
+                    $errName = 'E_ERROR';
+                break;    // 1
+                
+                case E_WARNING:
+                    $errName = 'E_WARNING';
+                    $ignoreError = $ignoreWarning;
+                break;  // 2
+                
+                case E_PARSE:
+                    $errName = 'E_PARSE';
+                break;    // 4
+                
+                case E_NOTICE:
+                    $errName = 'E_NOTICE';
+                break;   // 8
+                
+                case E_CORE_ERROR:
+                    $errName = 'E_CORE_ERROR';
+                break;   // 16
+                
+                case E_CORE_WARNING:
+                    $errName = 'E_CORE_WARNING';
+                    $ignoreError = $ignoreWarning;
+                break; // 32
+                
+                case E_COMPILE_ERROR:
+                    $errName = 'E_COMPILE_ERROR';
+                break;    // 64
+                
+                case E_COMPILE_WARNING:
+                    $errName = 'E_COMPILE_WARNING';
+                    $ignoreError = $ignoreWarning;
+                break;  // 128
+                
+                case E_USER_ERROR:
+                    $errName = 'E_USER_ERROR';
+                break;   // 256
+                
+                case E_USER_WARNING:
+                    $errName = 'E_USER_WARNING';
+                    $ignoreError = $ignoreWarning;
+                break; // 512
+                
+                case E_USER_NOTICE:
+                    $errName = 'E_USER_NOTICE';
+                    $ignoreError = $ignoreNotice;
+                break;  // 1024
+                
+                case E_STRICT:
+                    $errName = 'E_STRICT';
+                break;   // 2048
+                
+                case E_RECOVERABLE_ERROR:
+                    $errName = 'E_RECOVERABLE_ERROR';
+                break;    // 4096
+                
+                case E_DEPRECATED:
+                    $errName = 'E_DEPRECATED';
+                    $ignoreError = $ignoreDeprecated;
+                break;   // 8192
+                
+                case E_USER_DEPRECATED:
+                    $errName = 'E_USER_DEPRECATED';
+                    $ignoreError = $ignoreDeprecated;
+                break;  // 16384
             }
-        }
-
+            
+            if($ignoreError)
```

Comment:
here is the condition that generate errors on phpunit, if you change it to : 
```php
if($ignoreError || true)
```
with this condition we decide what to do with the error:
 - ignore
 - ignore and log
 - output and log

if we ignore throwing errors we go back to starting point, where we not have control over error output and when you open a modal and a server error happens browser will receive mixed json + visual html error : {success:false,message:'blabla html code'} <!doctype><html>...

i look further in log and code, to see if there is a solution or tragically some components depends on this "issue/bug".

Reply:
ok any suppressed in the code break the system like in jsSSE.php :
```php
if (@$_GET['event'] === 'sse') {
```

i look now if is only this